pages/page.py

```diff
@@ -32,8 +31,7 @@ def __init__(self, testsetup):
     @property
     def is_the_current_page(self):
         WebDriverWait(self.selenium, 10).until(lambda s: self.selenium.title)
-        Assert.equal(self.selenium.title, self._page_title,
-                     'Expected page title: %s. Actual page title: %s' % (self._page_title, self.selenium.title))
+        assert self._page_title == self.selenium.title
         return True
```

Comment:
You make a very good point. Let's leave it as you have it here, and I'll try to keep failure traceback in mind when converting asserts.